### DISCUSSION TITLE
Setting up Nginx Proxy in Host

### DISCUSSION THREAD

**Discussion by cdesouza71** [TandoorRecipes/recipes#1203] [Q&A]
Hello,

I'm trying to set up a Nginx proxy on the host to access the docker containers under http://my.local.host/menu. 

I have the following setup which does not work because for example the GET tries to load the resources from http://my.local.host/static/themes/select2-bootstrap-theme.d007f71f4d38.css which does not match the menu location.

compose.yml
` web_recipes:
    image: vabene1111/recipes
    restart: always
    env_file:
      - ./tandoor.env
    ports:
     - 1150:8080
`

Nginx Site
`
server {
  listen 80 default_server;
  listen [::]:80 default_server;
  root /var/www/html;
  index index.html index.htm index.nginx-debian.html;

  server_name _;

  location /menu/ {
    proxy_set_header Host $host;
    proxy_pass http://127.0.0.1:1150;
}
`

**Comment by vabene1111** [comment]
you cannot simply remove the nginx without taking care of static and media file serving. Please add the nginx back or handle those cases correctly in your own nginx. Also make sure to pass all headers correctly

**Reply by cdesouza71** [reply]
Hello,

thank you for your response. I took your advice and now the media files are being displayed correctly. The proxy however, still does not work. I will need to look at the headers again.
